Core/FixedValue.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace PCSharpGen.Core
+{
+    public class FixedValue : GameValue
```

Comment:
I think we don't need both "ComputedValue" and "FixedValue". They are all just "bonuses" that have a "calculation" bit, if the calculation is just "return 5" then its a fixed value, but I don't know if it needs an additional class for it.

Reply:
It will avoid a _lot_ of pointless method calls in the end, when we have to call it for hundreds of numbers on the character sheet, but maybe that's fine?

But does that mean every value carries a list of "other variables this might reference" and let the calculation choose one?  That's a lot of objects floating around...

Reply:
That would mean the fixed value would have a null (or empty) list of referenced variables, and then some null (or identity) computation callback.  That seems messier than just having two different types.

Reply:
There are also going to end up being Random values... which are something like "1d6 competence".  Which would force any generic type to have a "die size" that would be 1 most of the time.

Rendering them will also require a lot of knowledge about what "kind" of bonus it is.

Reply:
I think the bonuses could probably just be a function that accepts the character as a parameter, because thats what most of them end up being anyways.

Reply:
That will make it extremely hard to render them to the user in any useful way.   It'll just be "42" with no way to understand why.  And debugging would be similarly a nightmare. You'd just have huge arrays of delegate functions.  It would also be rather difficult to deal with a lot of the complex cases in some of the games (like you can't "replace" the same stat multiple times... how would we know if that had happened unless we have objects we can investigate that have the stat in them)?

I'm not sure I understand the objection to having them represented as objects... it seems to have several benefits and no downsides...

Reply:
Yea, if we can represent them as specific objects that would be useful I think. I don't know how that can be translated from the data files though.